src/components/ProductListScrollBar/ProductListScrollBar.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { ReactNode, useCallback } from 'react';
+import Scrollbars, { positionValues } from 'react-custom-scrollbars-2';
+
+type ScrollBarProps = {
+  className?: string;
+  children?: ReactNode;
+  onLoadMore?: () => void;
+  onScroll?: () => void;
+};
+
+const ProductListScrollBar = ({
+  children,
+  className = '',
+  onLoadMore,
+}: ScrollBarProps): JSX.Element => {
+  const handleScroll = (values: positionValues) => {
+    const { scrollTop, scrollHeight, clientHeight } = values;
+
+    const distanceFromBottom = scrollHeight - (scrollTop + clientHeight);
+    console.log(distanceFromBottom);
+
+    if (distanceFromBottom < 50) {
```

Comment:
I tested it with a few values and i thing something like `300` here has a better feel